modules/ocf_desktop/files/xsession/lightdm/lightdm-gtk-greeter.po

```diff
@@ -0,0 +1,3 @@
+#: ../src/lightdm-gtk-greeter.c:1479
+msgid "Incorrect password, please try again"
+msgstr "Incorrect username or password."
```

Comment:
how much space do we have in the UI? Wondering if we have enough to suggest creating an account or the password reset link. Maybe newlines work?